js/ai/src/generate/action.ts

```diff
@@ -141,7 +145,8 @@ async function generate(
   const request = await actionToGenerateRequest(
     rawRequest,
     tools,
-    resolvedFormat
+    resolvedFormat,
+    model.__action.metadata?.model as ModelInfo
```

Comment:
This is getting about to the limit of args before it should be an options object...

Reply:
done